codebasin/util.py

```diff
@@ -14,14 +14,15 @@
 import tomllib
 import typing
 from collections.abc import Iterable
+from io import TextIOWrapper
 from pathlib import Path
 
 import jsonschema
 
 log = logging.getLogger(__name__)
 
 
-def ensure_ext(path: os.PathLike[str], extensions: Iterable[str]):
+def ensure_ext(path: os.PathLike[str], extensions: Iterable[str]) -> None:
```

Comment:
I'm not opposed to this, but I'm curious.  Do conventions say to specify `-> None` in the case of no return types?  I'm not sure which style guides to look at here.

Reply:
My thought there was that Python functions always return `None`, even if they don't have a return statement. For the developer it seems to be better to be explicit, since then you remove the ambiguity of "did we leave out the return signature, or does it really return `None`"?

Seems like [mypy](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html#functions) recommends this as well, even if we don't use mypy.

Reply:
Okay, that's good enough for me!